index.js

```diff
@@ -456,6 +456,42 @@ require('yargs')
                 Logger.error(err.message);
             });
     })
+    .command('extract-decorators','extracts the decorator command sets and vocabularies from a list of model files', yargs =>{
+        yargs.demandOption('models','Please provide a model');
+        yargs.option('models', {
+            describe: 'The file location of the source models',
+            alias: 'model',
+            type: 'string',
+            array:true,
+        });
+        yargs.option('locale', {
+            describe: 'The locale for extracted vocabularies',
+            type: 'string',
+            default:'en'
+        });
+        yargs.option('removeDecoratorsFromSource', {
```

Comment:
Does this option have any effect?

Reply:
Its a flag used to determine whether to remove the decorators from source model.

Reply:
Sorry, I didn't see how it was passed into the Extractor.